src/Dropdown.js

```diff
@@ -4,6 +4,48 @@ import classNames from 'classnames';
 
 const DEFAULT_PLACEHOLDER_STRING = 'Select...';
 
+export const isValidLabelOrValue = value => (
+  /string|boolean|number/.test(typeof value));
+
+export const getOptionLabel = (option, label = option) => {
+  if (isValidLabelOrValue(option.label))
+    label = option.label;
+  else if (isValidLabelOrValue(option.value))
+    label = option.value;
+
+  return label;
+};
+
+export const getOptionValue = (option, value = option) => {
+  if (isValidLabelOrValue(option.value))
+    value = option.value;
+  else if (isValidLabelOrValue(option.label))
+    value = option.label;
+
+  return value;
+};
+
+export const Option = props => {
```

Comment:
move to own file

`import Option from './Option'`

Reply:
`import Option from './Option'` added